src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
-export interface Figure {
-
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
 }
+type Color = 'red' | 'green' | 'blue';
+type Callback = (arg: number) => number;
 
-export class Triangle {
+const callback: Callback = (n) => Math.floor(n * 100) / 100;
```

Comment:
Will be better create more meaningful variable name for this function to specify what it should do